### PR TITLE
[RG-171] Add GTKWave to cmake

### PR DESCRIPTION
**Pull request by skyler-rs**
> ### Motivate of the pull request
> - [x] To address an existing issue. If so, please provide a link to the issue: [RG-171]
> - [ ] Breaking new feature. If so, please describe details in the description part.

> ### Describe the technical details
> #### What is currently done? (Provide issue link if applicable)
> - nothing
>
> #### What does this pull request change?
> - GTKWave submodule cmake is now called by our main cmake

> ### Which part of the code base require a change
> 
> - [ ] Library: <Specify the library name>
> - [ ] Plug-in: <Specify the plugin name>
> - [ ] Engine
> - [ ] Documentation
> - [ ] Regression tests
> - [x] Continous Integration (CI) scripts

> ### Impact of the pull request
> - [ ] Require a change on Quality of Results (QoR)
> - [ ] Break back-compatibility. If so, please list who may be influenced.

### CODE REVIEW

File: third_party/gtkwave_cmake/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+project(GTKWave)
+
+if (CMAKE_BUILD_TYPE STREQUAL "Debug")
+    set(build_dir ${PROJECT_SOURCE_DIR}/../../dbuild)
+else ()
+    set(build_dir ${PROJECT_SOURCE_DIR}/../../build)
+endif()
+
+set(gtkwave_tar "gtkwave_linux_centos7.tar.gz")
+set(download_path "https://github.com/RapidSilicon/post_build_artifacts/releases/download/v0.1/${gtkwave_tar}")
```
Comment (KochynVolodymyr) [inline third_party/gtkwave_cmake/CMakeLists.txt:37]: isn't it private repo?

Comment (skyler-rs) [reply third_party/gtkwave_cmake/CMakeLists.txt:37]: I wondered the same thing, but it looks like the URL is public. I can dl it w/ a browser just by clicking the link https://github.com/RapidSilicon/post_build_artifacts/releases/download/v0.1/gtkwave_linux_centos7.tar.gz

Comment (ravic-rs) [reply third_party/gtkwave_cmake/CMakeLists.txt:37]: It's public. Public community can use that repo to download build artifacts. 
image

**Comment by skyler-rs** [comment]
Closing PR as it turned into a debug session, new PR can be found under #792

**Pull request closed without merging**